website/docs/general/contributor-guide.md

```diff
@@ -9,45 +9,41 @@ keywords:
 description: This article is a set of guidelines for Apache APISIX contributors, including things that a contributor can do and how to do it well.
 ---
 
-Please fee free to report bugs, submit suggestions, or submit PRs according to this guide.
+Please follow this guide to [report bugs](#submit-an-issue), [submit suggestions](#submit-an-issue), or [submit PRs](#submit-pr).
 
-## Submit an issue
+## Submitting an Issue
 
-1. Before submitting your issues, please go through a comprehensive search to make sure the problem cannot be solved just by searching.
+Before submitting an issue, try solving the problem through a comprehensive search.
 
-2. Check the Issue List to make sure the problem is not repeated.
+[Open issues](https://github.com/apache/apisix/issues) and the [discussions forum](https://github.com/apache/apisix/discussions) are good places to search for solutions before opening an issue.
 
-3. Create a new issue and choose the type of issue.
+1. Check the [open issues](https://github.com/apache/apisix/issues) to avoid duplication.
 
-4. Define the issue with a clear and descriptive title.
+2. Create a [new issue](https://github.com/apache/apisix/issues/new/choose) and choose the type of issue.
 
-5. Fill in necessary information according to the template.
+3. Define the issue with a clear and descriptive title.
 
-6. Choose a label after issue created.
+4. Fill in necessary information according to the template.
 
-7. Please pay attention to your issue, you may need provide more information during discussion.
+5. Choose proper label(s) after opening the issue.
```

Comment:
I asked other contributors to mock the process of opening an issue. It turns out that contributors are not able to change/select the labels by their own. I would suggest to delete this step for now.

We would definitely need to figure out the grainarities, accesses, and responsibilities here.
```suggestion

```

Reply:
Yes. I also thought so. Thank you for clarifying. Ultimately, we would want to have multiple templates and the templates should handle the labelling with minimal manual labelling from the maintainers.